src/repositories/indexable-repository.php

```diff
@@ -108,8 +108,8 @@ public function find_by_url( $url ) {
 
 		// Find by both url_hash and url, url_hash is indexed so will be used first by the DB to optimize the query.
 		return $this->query()
-					->where( 'url_hash', $url_hash )
-					->where( 'url', $url )
+					->where( 'permalink_hash', $url_hash )
```

Comment:
Is there a specific reason to change this to permalink_hash? Can you explain this in the technical choices. I also would rename the methods name into `find_by_permalink.` and rename `url` to `permalink` to have a clear function.

Reply:
At some point is was renamed to permalink instead of url. The function was broken so I fixed it, although the function is not used in the codebase.